src/app/user-navbar/user-navbar.component.ts

```diff
@@ -22,17 +25,34 @@ export class UserNavbarComponent implements OnInit {
   }
 
   goToHistory(): void {
-    this.router.navigateByUrl(`/flights/history/${this.customerId}`);
+    this.router.navigate([PathConstants.FLIGHT_HISTORY]);
   }
 
   goToUpcoming(): void {
-    this.router.navigateByUrl(`/flights/upcoming/${this.customerId}`);
+    this.router.navigate([PathConstants.FLIGHT_UPCOMING]);
   }
 
   ngOnInit(): void {
     if (!this.authService.isLoggedIn()) {
       this.router.navigate([PathConstants.LOGIN]);
     }
-    this.customerId = this.authService.userId;
+    this.determineActive();
+  }
+
+  determineActive(): void {
+    switch(this.parentComponent) {
+      case 'profile':
```

Comment:
should remove these raw string comparisons in case things change. A simple way to do this is to create a new `constants.js` file in `/environment/` that exports constant strings
```
export const PROFILE_SELECTION = "profile";
```
Then import this constant wherever it's used (presumably in the profile component and here). 

That is the javascript way of creating faux-enums. A better approach, since we're using TypeScript, is to create a [TS Enum](https://www.typescriptlang.org/docs/handbook/enums.html). 

```typescript
export enum NavbarSelection {
  PROFILE = "profile",
  HISTORY = "history"
  //  ... etc
}
```

Then use that in the switch:
```typescript
case NavbarSelection.PROFILE:
  // etc
```

Might also be worth considering making a function that changes the CSS dynamically, rather than using some booleans to determine what CSS field to use. This can be something to look into for future enhancements, but added as tech debt for now.